src/components/App.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+// Importing components
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+export default function App() {
+  return (
+    <div id="calculator-container" className="calculator-container">
+      <Display value="0" />
```

Comment:
- Don't set the default value here, set it the way it's done on the react documentation, kindly check [here](https://reactjs.org/docs/typechecking-with-proptypes.html#default-prop-values) on how to do so.